v1/descriptor_uuids.json

```diff
@@ -1,17 +1,17 @@
 [
-    { "name": "Characteristic Aggregate Format", "identifier": "org.bluetooth.descriptor.gatt.characteristic_aggregate_format", "uuid": "2905", "source": "gss" },
-    { "name": "Characteristic Extended Properties", "identifier": "org.bluetooth.descriptor.gatt.characteristic_extended_properties", "uuid": "2900", "source": "gss" },
-    { "name": "Characteristic Presentation Format", "identifier": "org.bluetooth.descriptor.gatt.characteristic_presentation_format", "uuid": "2904", "source": "gss" },
-    { "name": "Characteristic User name", "identifier": "org.bluetooth.descriptor.gatt.characteristic_user_name", "uuid": "2901", "source": "gss" },
-    { "name": "Client Characteristic Configuration", "identifier": "org.bluetooth.descriptor.gatt.client_characteristic_configuration", "uuid": "2902", "source": "gss"},
-    { "name": "Environmental Sensing Configuration", "identifier": "org.bluetooth.descriptor.es_configuration", "uuid": "290B", "source": "gss"},
-    { "name": "Environmental Sensing Measurement", "identifier": "org.bluetooth.descriptor.es_measurement", "uuid": "290C", "source": "gss"},
-    { "name": "Environmental Sensing Trigger Setting", "identifier": "org.bluetooth.descriptor.es_trigger_setting", "uuid": "290D", "source": "gss"},
+    { "name": "Characteristic Extended Properties", "identifier": "org.bluetooth.descriptor.characteristic.extended_properties", "uuid": "2900", "source": "gss" },
```

Comment:
Some of the changes seem a bit inconsistent or are you introducing a new pattern?
`org.bluetooth.descriptor.characteristic.user_name`: should be `characteristic_user_name`?
`org.bluetooth.descriptor.client_characteristic.configuration`: should be `client_characteristic_configuration`?
Etc.

Reply:
You're right - sorry. Got carried away.